src/scenegraph/Model.h

```diff
@@ -207,6 +211,10 @@ namespace SceneGraph {
 		RefCountedPtr<Graphics::VertexBuffer> m_aabbVB;
 		RefCountedPtr<Graphics::Material> m_aabbMat;
 		Graphics::RenderState *m_state;
+
+		// Vector with mounts used by guns
+		GunMounts m_mounts;
```

Comment:
I'd suggest that the list of gun mounts should be part of ShipType or something like that, instead of part of a Model. Semantically, the only part of a gun mount that's part of a model is the tag used for positioning the mounted item. This is okay as-is, but please include a `//TODO` to this effect so I can improve it later.

Reply:
I left a comment (there were a comment also on `FixedGuns` thought...), but if I may, I would say that even if I think a merge of data of `ShipType` and `GunMount` is needed, I postpone a merge of them until some steps toward turrets (or a kind of external "attachement points") is made.